javascript-data-model/order-history.js

```diff
@@ -0,0 +1,68 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4th, 2020',
+    totalCost: 34.00,
+    shipRecipient: 'JS Masher',
+    orderNumber: '# 114-3941689-8772232',
+    deliveryDate: 'August 8th, 2020',
+    returnWindow: 'Septemeber 7th, 2020',
+    itemDescription: [{
+      type: 'book',
+      title: 'javaScript for impatient programmers',
+      author: 'Dr. Axel Raschmayer',
+      price: 31.55
+    }]
+  },
+  {
+    orderPlaced: 'July 19th, 2020',
+    totalCost: 44.53,
+    shipRecipient: 'JS Masher',
+    orderNumber: '# 113-9984268-1280257',
+    deliveryDate: 'July 20th, 2020',
+    returnWindow: 'August 19th, 2020',
+    itemDescription: [{
+      type: 'book',
+      title: 'The Timeless Way of Building',
+      author: 'Christopher Alexander',
+      price: 41.33
+    }]
+  },
+  {
+    orderPlaced: 'July 4th, 2020',
+    totalCost: 17.22,
+    shipRecipient: 'JS Masher',
+    orderNumber: '# 114-2875557-9059409',
+    deliveryDate: 'July 7th, 2020',
+    returnWindow: 'August 5th, 2020',
+    itemDescription: [{
+      type: 'Gamecube Controller adapter',
```

Comment:
This is a good way of doing this, but to stay consistent with your object for books, I would make this something like `type: "electronic"`, and then use the `title` property to hold the actual name of the item